src/main/java/city/parking/services/ParkingProcessService.java

```diff
@@ -0,0 +1,118 @@
+package city.parking.services;
+
+import city.parking.entities.Money;
+import city.parking.entities.ParkingProcess;
+import city.parking.entities.ParkingProcessPartialUpdateRequest;
+import city.parking.exceptions.ParkingMeterAlreadyInUseException;
+import city.parking.exceptions.ParkingMeterNotRunningException;
+import city.parking.exceptions.ParkingProcessNotFoundException;
+import city.parking.repositories.ParkingProcessRepository;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static city.parking.services.ParkingCostCalculationUtil.calculatePrimaryParkingCost;
+
+@Service
+public class ParkingProcessService {
+
+    private final ParkingProcessRepository parkingProcessRepository;
+
+    public ParkingProcessService(ParkingProcessRepository parkingProcessRepository) {
+        this.parkingProcessRepository = parkingProcessRepository;
+    }
+
+    public List<ParkingProcess> findAll() {
+        return parkingProcessRepository.findAll();
```

Comment:
naprawdę ktoś będzie chciał wyciągać wszystkie? po co?

Reply:
Racja, do usunięcia, przydatne to było do sprawdzania czy wszystko działa zgodnie z przewidywaniami w miarę pisania (docelowo sprawdzać to powinien test)